app/renderer/components/frame/frame.js

```diff
@@ -750,8 +750,6 @@ class Frame extends React.Component {
       if (!this.frame.isEmpty()) {
         windowActions.setNavigated(e.url, this.props.frameKey, false, this.props.tabId)
       }
-      // force temporary url display for tabnapping protection
-      windowActions.setMouseInTitlebar(true)
```

Comment:
I tested both with and without this forced display and as far as I can tell no visual changes. In both cases, setting page to auto-refresh removes titleMode temporarily and letting tab idle while navigating other tab didn't make any action. Please let me know if I should test another scenario for tabnapping.